gcp/src/main/java/org/apache/iceberg/gcp/GCPProperties.java

```diff
@@ -54,7 +63,11 @@ public class GCPProperties implements Serializable {
   private String gcsOAuth2Token;
   private Date gcsOAuth2TokenExpiresAt;
 
-  public GCPProperties() {}
+  private int gcsDeleteBatchSize;
+
+  public GCPProperties() {
+    gcsDeleteBatchSize = GCS_DELETE_BATCH_SIZE_DEFAULT;
```

Comment:
probably ok to set this here, alternative maybe just in the line above? `private int gcsDeleteBatchSize = GCS_DELETE_BATCH_SIZE_DEFAULT`

Reply:
Sure, I updated this. I was just following the pattern in `S3FileIO`.